core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -57,66 +57,77 @@ var LearnerDashboardPage = function() {
     await waitFor.elementToBeClickable(
```

Comment:
Remove this check as we already do this inside the `action.click` method!

Reply:
Done